src/main/java/org/folio/consortia/utils/ErrorHelper.java

```diff
@@ -33,13 +34,23 @@ public static Errors createInternalError(String message, ErrorCode errorCode) {
   }
 
   public static Errors createExternalError(String message, ErrorCode errorCode) {
-    return createErrors(createError(message, ErrorType.FOLIO_EXTERNAL_OR_UNDEFINED, errorCode));
+    return createErrors(createError(message, ErrorType.EXTERNAL, errorCode));
+  }
+
+  public static Errors createPermissionError(FeignException e, ErrorCode errorCode){
+    String message = e.getMessage();
+    String extractedMessage = extractPermissionFromErrorMessage(message);
+    return createErrors(createError(extractedMessage, ErrorType.INTERNAL, errorCode));
+  }
+
+  private String extractPermissionFromErrorMessage(String e) {
+    return e.substring(e.lastIndexOf('[') + 1, e.lastIndexOf(']')); // extract the main part from the error message
   }
 
   public enum ErrorType {
-    INTERNAL("-1"),
-    FOLIO_EXTERNAL_OR_UNDEFINED("-2"),
-    EXTERNAL_OR_UNDEFINED("-3"),
+    EXTERNAL("-1"), // bad request or client error
```

Comment:
-1 error for client error. for example bad request or illegal argument exception
-2 error for internal error. for example error with db or not connect with other module